### ISSUE TITLE
Provide mapping from a debian distros versions and codenames

### ISSUE THREAD

**Issue by andrewschoen** [ceph/chacra#187]
For example, these two urls should provide the same output:

https://chacra.ceph.com/binaries/ceph/jewel/HEAD/ubuntu/xenial/ https://chacra.ceph.com/binaries/ceph/jewel/HEAD/ubuntu/16.04/

I don't think we care about this for rpm-based distros.

**Comment by andrewschoen** [comment]
I think we'd want to actually store ``distro_version`` and ``distro_codename`` on the Binary and Repo models as well.

**Comment by dmick** [comment]
It could be useful for RPM as well; RHEL seems to actually use codename, and if it can be supported by the same mechanism without huge effort.  CentOS seems to be the standout.  No idea about others (Fedora?....SUSE?...)

**Comment by alfredodeza** [comment]
I don't think we need to provide the naming shim for chacra. Everything/anything that will be consumed is already provided by shaman. URLs in chacra could very well be a url using a short/unique sha1 (bit.ly style) and it would still be OK.

**Comment by andrewschoen** [comment]
Yeah, I can see the point in only doing this in Shaman as it's going to be what people are going to use to query for the status of repos, not Chacra.
